front/src/utils/date.ts

```diff
@@ -184,5 +202,29 @@ export function serializeDateTimeWithoutYear(date: Date): string {
   return dayjsDate.format('DD/MM HH:mm:ss');
 }
 
-export const isoUtcStringToLocaleDateString = (isoUtcString: string) =>
-  dayjs(isoUtcString).local().format();
+/**
+ * Convert an ISO date into a string formatted as 'DD/MM/YYYY' and extract the numeric values for hours and minutes.
+ * @param {string} arrivalTime - Arrival time at which the train should arrive at the location. (Format: 'YYYY-MM-DDTHH:mm:ss+HH:mm')
+ * @returns {object} An object containing the parsed date and time.
+ */
+export function extractDateAndTimefromISO(arrivalTime: string) {
```

Comment:
From what I discussed last week with Florian, it would be better to convert the string into a date and then to use the librairies to extract the date, the time etc. You can have a look to the function formatDateToString